chapter-00-Prelude-to-War.md

```diff
@@ -0,0 +1,316 @@
+---
+title: "Chapter 00: Prelude to War"
+date: 2018-05-13
+---
+
+
+# Prelude to War
```

Comment:
Remove. Again, you do not need this; please look at how the rest of the site is put together and emulate it.

Reply:
And also this entire file is both a duplicate *and in the wrong location.* Delete it.